### DISCUSSION TITLE
testing the redesigned wg-easy

### DISCUSSION THREAD

**Discussion by this-xkit** [wg-easy/wg-easy#1490] [Q&A] [answered]
Hi everyone,

I wanted to ask if it's possible to try out the redesigned version of wg-easy at this stage, or is it still too early for that?

I noticed that the standard installation instructions are provided in the feat-ui branch, but I encountered some difficulties while attempting to install it on my own. If anyone has insights on the current status or any tips for a successful installation, I would greatly appreciate your input!

Thank you!

**Comment by kaaax0815** [comment, marked as answer]
You can use the `development` tag. It's not pointing to the latest commit all the time tho. And it's still fairly early
